.nycrc

```diff
@@ -1,6 +1,13 @@
 {
-  "extends": "node_modules/ocular-dev-tools/templates/.nycrc",
+  "all": "true",
```

Comment:
Nyc seems to support js config. Maybe we provide a template in ocular-dev-tools?

https://github.com/istanbuljs/nyc#configuration-files

Reply:
I can play with it... What I have found is that the `extends` does not work as expected - any overrides (`include`/`exclude`) are ignored.